dataeng/resources/opsgenie-enable-heartbeat.sh

```diff
@@ -25,7 +25,23 @@ if [ -n "$OPSGENIE_HEARTBEAT_NAME" ] && \
     HEARTBEAT_API_PING_URL="$HEARTBEAT_API_URL/ping"
     ALERT_MESSAGE="Heartbeat [$OPSGENIE_HEARTBEAT_NAME] expired - job is likely stuck."
     JSON_REQ_DATA="{ \"interval\": \"$OPSGENIE_HEARTBEAT_DURATION_NUM\", \"intervalUnit\": \"$OPSGENIE_HEARTBEAT_DURATION_UNIT\", \"alertMessage\": \"$ALERT_MESSAGE\", \"enabled\": true }"
```

Comment:
Nit: IMO, this `JSON_REQ_DATA` variable definition should be next to the next section after the heartbeat is created.